### ISSUE TITLE
1.1 7.1: Console Errors

### ISSUE THREAD

**Issue by mesl** [UltimateGamer200/UltraColor#15]
Finding these console errors, Unsure what caused them exactly:
https://pastebin.com/ysumTkYU

Hope you can fix this!
Thank you

**Comment by UltimateGamer200** [comment]
Yeah, I can't seem to pinpoint a cause for this. In fact, I can't even seem to reproduce it in my testing environment. Would you mind maybe sending me your setup for me to test out? I would need your UltraColor folder inside your plugins folder which contains your settings file, data.db, etc (just zip it up and upload the zip file here). **If you're using the MySQL Database feature, I highly recommend you censor your database details so that your database is secure.**

Also, do you know when exactly these errors occurred (like what's the command you or a player typed in that caused that error trace)?

**Comment by UltimateGamer200** [comment]
Hmm, I still can't reproduce this bug. No console errors are given and it works just fine. Even tried regenerating the data.db file, adding some debug, and trying the command you gave, this bug still can't be reproduced. I might have to try your UltraColor setup to see if that can reproduce this bug. If not, I don't think there's much else I can do until you try the next update.

Could you send me your setup for me to test out? This setup would be your UltraColor folder in a zip file. **Be sure to censor your database details if you're using the MySQL feature of the plugin so that your database remains secure from people who want to hack it. Censoring the details simply means replacing things like your database password, user, name, host, etc, with an * inside the zip file.**

**Comment by UltimateGamer200** [comment]
Interesting. When I tried your setup, those errors were finally shown. After some testing, I believe I found a fix. This fix involved a new data value for players for the data.db file called, "Player_Name". This value is automatically changed when a player joins. Just make sure for players that are stored when you try to use commands on them, they have that value stored. There is a way to manually add it in, but the file is machine-generated, so it may be risky without my guidance. 

Anyway, this fix will be in the next update, but in the meantime, feel free to test it with a test jar [here ](https://ufile.io/d5xzvlba) and let me know if it's fixed!

**Comment by mesl** [comment]
Seems to be fixed! Thank you very much!
